container/container.go

```diff
@@ -54,7 +54,7 @@ type Option func(*DockerContainer)
 // New creates a new Container with given options.
 func New(options ...Option) (*DockerContainer, error) {
 	c := &DockerContainer{
-		callTimeout: 10 * time.Second,
+		callTimeout: 60 * time.Second,
```

Comment:
I changed this  value because some test always reach this timeout.
Anyone know why with have this timeout setup in the first place?

Reply:
Probably to reduce default timeout which may be from 180 up to 360 sec as I remember

Reply:
Actually it makes a lot of test failed or timeout... what is happening?